general/v0.1/introduction.md

```diff
@@ -2,46 +2,46 @@
 
 ## 01: Introduction
 
-This document introduces the concept of "Fediverse Auxiliary Service
-Providers", external services that fediverse server instances can use to
-perform a variety of tasks. It defines the way such providers are set-up
-within an instance's server software and how the two communicate.
+This document introduces the concept of Fediverse Auxiliary Service
+Providers (FASPs), third-party services that fediverse server software can use to
+perform a variety of tasks, and defines the way such FASPs are configured
+within fediverse server software and how the two communicate.
```

Comment:
While writing this I sometimes struggled with a distinction that I think might be important to make:

1. fediverse server software will need to implement these specs for the whole provider thing to work
2. instance operators can choose if, when and how to make use of whatever providers they like (and their instance's software supports)

So I think it is actually the instance that "can use [the FASP] to perform a variety of tasks".

Also, did you remove the word "instance" here, because we only introduce it later?

I would still be fine with these changes, just wanting to explain my reasoning. It may also explain some inconsistencies in the other files :grimacing:

Reply:
My thinking is that client apps that are not "instances" might also make use of a FASP, two examples that spring to mind are a client app that wants to query IFTAS for a filter list of reserved usernames (slurs etc), and Wordpress is interested in a domain label query for individual Wordpress sites, neither of which fall in to the traditional "instance" rubrik, but both could be capable of and likely interested in using the FASP API.